### PR TITLE
Add semicolon at the end of the function prototypes

### PR DESCRIPTION
**Pull request by InfoHunter**
I think these semicolons should be added to match the C language syntax...

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/EVP_sm3.pod [hunk 1]
```diff
@@ -42,8 +42,8 @@ L<EVP_DigestInit(3)>
 
 =head1 COPYRIGHT
 
-Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
-Copyright 2017 Ribose Inc. All Rights Reserved.
+Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2017-2018 Ribose Inc. All Rights Reserved.
```
Comment (mattcaswell) [inline doc/man3/EVP_sm3.pod]: Drop the change in copyright. The Ribose claim only relates to the original submission, so doesn't get extended by later modifications. The OpenSSL claim will get updated automatically as part of the release process (we have scripts to do this).

File: doc/man3/EVP_sm4_cbc.pod [hunk 2]
```diff
@@ -52,8 +52,8 @@ L<EVP_CIPHER_meth_new(3)>
 
 =head1 COPYRIGHT
 
-Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
-Copyright 2017 Ribose Inc. All Rights Reserved.
+Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2017-2018 Ribose Inc. All Rights Reserved.
```
Comment (mattcaswell) [inline doc/man3/EVP_sm4_cbc.pod]: Same here.

File: doc/man3/EVP_sm3.pod (hunk 1 above)
Comment (InfoHunter) [reply doc/man3/EVP_sm3.pod]: Okay, understood. Will fix that later

**Comment by InfoHunter** [comment]
Address the date problem.

**Comment by InfoHunter** [comment]
I will merge this soon

**Comment by InfoHunter** [comment]
Merged as 21ebd2f , closing...

**Pull request closed without merging**